packages/zoe/src/zoeService/zoe.js

```diff
@@ -73,7 +73,7 @@ const makeZoeKit = (
   },
   meteringConfig = {
     incrementBy: 25_000_000n,
-    initial: 50_000_000n, // executeContract for treasury is 13.5M
+    initial: 75_000_000n, // executeContract for treasury is 13.5M
```

Comment:
@katelynsills This was enough of a boost to clear CI. More boost might be necessary to preserve more overhead. The comment is probably stale now. Do we have a tool for measuring computron cost? cc @warner

Reply:
@kriskowal does it seem plausible that zip archives are 50% more compute-intensive? What is it doing that take so much more computrons?

Reply:
The `nestedEvaluate` format is a `string`, whereas the `endoZipArchive` is a `string` in base64, which we then convert to a TypedArray with `Base64.decode` (which is probably free regardless of size since it’s native and I assume not metered), but then we slice the TypedArray into small files and UTF-8 decode them (which might be metered, but should be close to free on XS).

Some of this overhead should turn in our favor when we switch CapTP to a binary protocol, which will reduce the size of setBundle messages by about 25% and eliminate the base64 decode. The transcode might not change.

The `nestedEvaluate` format is a code string that contains code strings. The link phase is expressed as code, whereas with `endoZipBase64`, it’s expressed as a JSON blob that creates a Compartment graph and links them. That’s overhead. The overhead could probably be reduced by translating the JSON compartment map into code.

There would remain some overhead though because `nestedEvaluate` is a single-compartment runtime and `endoZipBase64` puts us on footing for LavaMoat style dependency sandboxing.

Reply:
ah. right. Compartment-per-package for least-authority linkage is why we would pay this cost.

I wonder whether `Base64.decode` should be metered... or if all uses of it are going to cost enough to build the input so that it doesn't matter. I probably should open an issue.

Reply:
Opening an issue sounds good to me. Where does Moddable want us to open issues?

Reply:
it should be an agoric-sdk issue.

We probably don't need help from Moddable. We should be able to just combine the native base64 implementation that does something O(n) on the length of the input... for example, since regexps are already metered: `s.match(/.*/)`

Reply:
Excellent. Filed https://github.com/Agoric/agoric-sdk/issues/3874